### PR TITLE
feat: 🎸 Updated the link for Storage Guide in the Services page

### PR DESCRIPTION
**Pull request by yangliu2009**
Updated the link to
https://ithelp.brown.edu/kb/articles/decide-how-to-store-and-share-files-for-researchers
for Storage Guide

### Pull Request
---

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x ] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [ x] Commitizen was used for all commits.
- [x ] Local site looks as expected after changes.
- [ x] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Comment by yangliu2009** [comment]
@fernandogelin @mirestrepo - Is there anything needed from me for the pull request? Thanks.

**Comment by mirestrepo** [comment]
The link looks good to me, @fernandogelin should @yangliu2009 also remove [this page](https://github.com/brown-ccv/ccv-website/blob/master/content/english/services/infrastructure/storage.md)? And are there more hidden related content?

File: themes/gb-theme/layouts/_default/list.html [hunk 1]
```diff
@@ -59,7 +59,7 @@ <h1 class="mx-4 text-secondary">{{ humanize $key }}</h1>
               
                 <h3>{{ markdownify .Site.Data.file_storage.lead }}</h3>
                 <hr class="bg-primary">
-                <a class="fs-26" href="{{.Site.BaseURL}}/{{.Site.Data.file_storage.path}}">
+                <a class="fs-26" href="https://ithelp.brown.edu/kb/articles/decide-how-to-store-and-share-files-for-researchers">
```
Comment (fernandogelin) [inline themes/gb-theme/layouts/_default/list.html]: @yangliu2009 we should keep the path in the yml file, just change this line to: `<a class="fs-26" href="{{.Site.Data.file_storage.path}}">`
and update the `path` in the `data/file_storage.yaml`

**Review by fernandogelin — CHANGES_REQUESTED** [review]

**Comment by fernandogelin** [comment]
> The link looks good to me, @fernandogelin should @yangliu2009 also remove [this page](https://github.com/brown-ccv/ccv-website/blob/master/content/english/services/infrastructure/storage.md)? And are there more hidden related content?

yes, I think that page can be removed.

**Comment by yangliu2009** [comment]
@fernandogelin @mirestrepo - Made changes as requested except that {{.Site.BaseURL}}/ is not added, since it resulted in 404 on my local computer.

**Comment by yangliu2009** [comment]
@fernandogelin @mirestrepo - Can the pull request be approved? Thanks.

Yang

**Review by fernandogelin — APPROVED** [review]

**Review by paulstey — APPROVED** [review]

**Pull request merged**